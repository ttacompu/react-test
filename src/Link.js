import React from 'react';

class Link extends React.Component{
    constructor(props){
       super(props);
    }

    render(){
        return ( <a className="test" href={this.props.address}>Click</a>)
    }

}
export default Link;