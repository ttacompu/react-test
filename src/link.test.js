import Link  from './link';
import React from 'react';
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json'


  describe('<Link />', () => {
    it('Should contain link with test', () => {
      const tree=mount(<Link address='http://www.google.com' />)
      expect(toJson(tree)).toMatchSnapshot();
      tree.unmount();
    });
  });