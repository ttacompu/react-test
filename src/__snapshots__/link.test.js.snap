// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> Should contain link with test 1`] = `
<Link
  address="http://www.google.com"
>
  <a
    className="test"
    href="http://www.google.com"
  >
    Click
  </a>
</Link>
`;
